src/Raven.Server/Utils/Cli/RavenCli.cs

```diff
@@ -747,10 +748,17 @@ private static bool CommandReplaceClusterCert(List<string> args, RavenCli cli)
             }
 
             cli._server.ServerStore.EnsureNotPassive();
+
+            // This restriction should be removed when updating to .net core 2.1 when export of collection is fixed.
+            // With export, we'll be able to load the certificate and export it without a password, and propogate it through the cluster.
+            if (PlatformDetails.RunningOnLinux && string.IsNullOrWhiteSpace(password) == false)
+                throw new NotSupportedException("Replacing the cluster certificate in Linux does not support password protected certificates.");
```

Comment:
Why isn't this blocked on Windows as well, then?

Reply:
Sorry, missed it